bot/exts/evergreen/cowsay.py

```diff
@@ -0,0 +1,64 @@
+from random import choice
+
+from cowsay import get_output_string
+from discord import Embed
+from discord.ext import commands
+
+from bot.constants import Colours, NEGATIVE_REPLIES
+
+
+class Cowsay(commands.Cog):
+    """Cog for the cowsay command."""
+
+    def __init__(self, bot: commands.Bot):
+        self.bot = bot
+
+    @commands.command(aliases=["cow"])
+    async def cowsay(self, ctx: commands.Context, character: str = "Cow", *, text: str = "I'm a Cow") -> None:
```

Comment:
I don't think we should keep `text` a default value **in the parameter** as when one specifies the character to be a `Dragon`, the text which should come is `I'm a Dragon`, but here it would be `I'm a Cow`.

One possible solution is to keep the `text` `None`, and then check using a if statement, and if it is `None`, we can put in the default string
```py
if not text:
  text = f"I'm a {character}"
```